code/modules/research/rdconsole.dm

```diff
@@ -833,24 +829,40 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 				design_list["materials"] = materials_list
 				design_list["id"] = D.id
 				design_list["name"] = sanitize(D.name)
-				var/check_materials = 1
+				var/c = 1

```

Comment:
I may not be a good programmer, but I think it will reduce any confusion if you use actual words as variables instead of letters.